### PR TITLE
Terra assets PR v1

### PR DESCRIPTION
**Pull request by 0xchain1**

### CODE REVIEW

**Comment by specialist0x** [comment]
We need to add tokens to the /tokens repo:
image

File: src/state/stake/stake-constants.ts [hunk 1]
```diff
@@ -279,8 +290,51 @@ const AURORA_POOLS: StakingTri[] = [
...
-    chefVersion: ChefVersions.V2
+    chefVersion: ChefVersions.V2,
+    doubleRewards: true
   },
+  {
+    ID: 9,
+    poolId: 2,
+    tokens: [WNEAR[ChainId.AURORA], LUNA[ChainId.AURORA]],
+    stakingRewardAddress: MASTERCHEF_ADDRESS_V2[ChainId.AURORA],
+    lpAddress: '0xd1654a7713617d41A8C9530Fb9B948d00e162194',
```
Comment (baboobhaiya) [inline src/state/stake/stake-constants.ts:301]: this is LP address of already active pool: https://github.com/trisolaris-labs/apr/blob/master/apr.py#L95

File: src/fetchers/farms.ts [hunk 2]
```diff
@@ -2,7 +2,7 @@ import useSWR from 'swr';
 import { ExternalInfo } from '../state/stake/stake-constants';
 
 async function fetcher() {
-    const response = await fetch('https://raw.githubusercontent.com/trisolaris-labs/apr/master/datav2.json')
+    const response = await fetch('https://raw.githubusercontent.com/trisolaris-labs/apr/master/datatest.json')
```
Comment (0xchain1) [inline src/fetchers/farms.ts:5]: TODO - change to correct url once ready

File: src/state/stake/user-farms.ts [hunk 3]
```diff
@@ -26,7 +26,7 @@ export function useSingleFarm(version: string): StakingTri[] {
 
 
   useEffect(() => {
-    fetch('https://raw.githubusercontent.com/trisolaris-labs/apr/master/datav2.json')
+    fetch('https://raw.githubusercontent.com/trisolaris-labs/apr/master/datatest.json')
```
Comment (0xchain1) [inline src/state/stake/user-farms.ts:29]: TODO - change to correct url once ready

**Comment by 0xchain1** [comment]
- Added new pools (see below)
- Tested deposit/withdraw
- double reward token hiding
- TODO - change url to datav2.json before merging
- TODO - remove USD values and hide APRs


wNEAR <> atLuna

Tested Deposit

[https://explorer.mainnet.aurora.dev/tx/0x6b135b7da7506120eff68b2e113637484f7ee05869843da76e4cc75807ba946d/token-transfers](https://explorer.mainnet.aurora.dev/tx/0x6b135b7da7506120eff68b2e113637484f7ee05869843da76e4cc75807ba946d/token-transfers)

Tested Withdraw 

[https://explorer.mainnet.aurora.dev/tx/0x39c38e86f24a0bc8aa6ddfaf0d41562e136d638473d7cb9ad279c35ca875e3d5/token-transfers](https://explorer.mainnet.aurora.dev/tx/0x39c38e86f24a0bc8aa6ddfaf0d41562e136d638473d7cb9ad279c35ca875e3d5/token-transfers)

wNEAR <> atUST

Tested Deposit

[https://explorer.mainnet.aurora.dev/tx/0x3821a773f28a8834bc7b8d0cc1e8b1ce190d2e3adefb8ce1300f978f3c8efd93/token-transfers](https://explorer.mainnet.aurora.dev/tx/0x3821a773f28a8834bc7b8d0cc1e8b1ce190d2e3adefb8ce1300f978f3c8efd93/token-transfers)

Tested Withdraw 

[https://explorer.mainnet.aurora.dev/tx/0xf33d6827f3b834f3d743b0d1cb535e0300f65e79a0c0f8d83fe33b365f3f0017/token-transfers](https://explorer.mainnet.aurora.dev/tx/0xf33d6827f3b834f3d743b0d1cb535e0300f65e79a0c0f8d83fe33b365f3f0017/token-transfers)

**Pull request closed without merging**